scripts/deploy.ts

```diff
@@ -79,6 +79,10 @@ async function main() {
       walletImplementationContractName = 'ZkethWalletSimple';
       contractPath = `contracts/coins/${walletImplementationContractName}.sol:${walletImplementationContractName}`;
       break;
+    case 80084:
+      walletImplementationContractName = 'BeraWalletSimple';
```

Comment:
It should be WalletSimple

Reply:
Also need to add 
 forwarderContractName = 'ForwarderV4';
 forwarderFactoryContractName = 'ForwarderFactoryV4';

Reply:
Thanks, i followed the zksync so might have missed these